include/fix8/thread.hpp

```diff
@@ -67,7 +67,7 @@ class _f8_threadcore
 	pthread_attr_t _attr;
 	pthread_t _tid;
 #elif (FIX8_THREAD_SYSTEM == FIX8_THREAD_STDTHREAD)
-	std::unique_ptr<std::thread> _thread;
+	std::atomic<std::thread*> _thread_ptr;
```

Comment:
why not use std::unique_ptr in here?

Reply:
_thread_ptr is non-null if there's a running thread.  It's tested simultaneously across two threads, so it doubles as a lock-free test.

Reply:
how does that relate to my question about unique_ptr?
anyway std::atomic requires type to be trivially copyable - so std::atomic<std::unique_ptr> i s ano go.